File: App-Revistas/src/app/Objetos/Cliente.ts
import { Perfil } from "./Perfil";
import { Usuario } from "./Usuario";

export class Cliente {

    usuario: Usuario;
    perfil: Perfil;

    constructor(
        usuario: Usuario,
        perfil: Perfil
    ) {
        this.usuario = usuario;
        this.perfil = perfil;
    }
}

File: BackendRevistas/src/main/java/Entidades/Cliente.java
package Entidades;

/**
 *
 * @author marco
 */
public class Cliente {

    private Usuario usuario;
    private Perfil perfil;

    public Cliente(Usuario usuario, Perfil perfil) {
        this.usuario = usuario;
        this.perfil = perfil;
    }

    public Usuario getUsuario() {
        return usuario;
    }

    public void setUsuario(Usuario usuario) {
        this.usuario = usuario;
    }

    public Perfil getPerfil() {
        return perfil;
    }

    public void setPerfil(Perfil perfil) {
        this.perfil = perfil;
    }
}
